src/SessionInstance.php

```diff
@@ -188,13 +198,17 @@ public function destroy()
     {
         $this->init();
 
-        # Start the session up, but ignore the error about headers already being sent
-        @session_start();
```

Comment:
yes, the recommended way is to destroy the session and remove the cooke - as it is in my branch. without deleting the cookie session_destroy seems to be useless. I will write another test to see if session_start and session_destroy is required or not.